clion-plugin/src/main/kotlin/org/utbot/cpp/clion/plugin/client/ClientManager.kt

```diff
@@ -23,7 +22,7 @@ class ClientManager(val project: Project): Disposable {
     }
 
     private fun subscribeToEvents() {
-        with(ApplicationManager.getApplication().messageBus.connect()) {
```

Comment:
Here was the cause for exception: after closing project this listeners was not disposed, because it had application lifetime